README.md

```diff
@@ -1,2 +1,42 @@
 # CSCI 420 (Spring 2021) MontyPython UML Editor
 
+This command line program creates a text representation of a UML diagram though user input. The program’s capabilities include adding, deleting, displaying, and modifying UML diagram elements. It also supports the ability to save and load diagrams.
+
+## Prerequisites
+1. To run the program, [Python 3.9.1](https://www.python.org/downloads/) needs to be installed.
```

Comment:
Change the numbers so not everything is step 1.

Reply:
That's GitHub markdown syntax for ordered lists

Reply:
> That's GitHub markdown syntax for ordered lists

Ah I see now, my bad.